### PR TITLE
OCP-11286: change iptables flush to be more atomic

### PR DESCRIPTION
**Pull request by rbbratta**
Try to fix OCPQE-2116

### CODE REVIEW

**Comment by rbbratta** [comment]
https://issues.redhat.com/browse/OCPQE-2116

**Comment by rbbratta** [comment]
http://file.rdu.redhat.com/~rbrattai/logs/cucushift-2020-09-10-21-49-41.log

File: features/step_definitions/networking.rb [hunk 1]
```diff
@@ -231,15 +231,23 @@
...
+  # Flush all the chains first, then delete to prevent reference issues
+  # There must be no references to the chain.
+  # If there are, you must delete or replace the referring rules before the chain can be deleted.
+  # The chain must be empty, i.e. not contain any rules.
+  delete_chain = tables.map { |table|
+    "iptables -t #{table} -X"
+  }.join(" ; ")
+  @result = _host.exec(flush_chain + " ; " + delete_chain)
+  # ignore any chain delete errors for now and just see if it triggers the iptableSync
+  # raise "failed to delete iptables chain #{table}" unless @result[:success]
```
Comment (liangxia) [inline features/step_definitions/networking.rb:248]: The comment is not accurate now. Needs update, too.

**Pull request merged**